fn_grpc_interface/fn_grpc_interface/components/function_grpc.py

```diff
@@ -0,0 +1,290 @@
+                _grpc_secure_connection = config_file_data[1]
+                _grpc_communication_type = config_file_data[0]
+                _grpc_google_tocken = config_file_data[2]
+                _grpc_certificate_path = config_file_data[2]
+                _grpc_interface_file_dir = self.options.get('interface_dir')
+
+                log.info("gRPC Package Name : {} gRPC Stub Method Name : {} gRPC Request Class Name : {}" \
+                         .format(_grpc_package_name, _grpc_rpc_stub_method_name, _grpc_request_method_name))
+            except Exception as e:
+                yield StatusMessage("failed to parse the function parameters.{}".format(e))
+                raise FunctionError(e)
+
+            # Importing gRPC proto buff files from the given interface directory.
+            try:
+                module_files = self._get_interface_file_names(_grpc_interface_file_dir, _grpc_package_name)
+                grpc_interface_module_list = self._get_grpc_interface_module(module_files, _grpc_interface_file_dir,
+                                                                             _grpc_package_name)
+                logging.debug(grpc_interface_module_list)
+
+                grpc_stub_tuple = self._get_grpc_class(grpc_interface_module_list, 'stub')
+                grpc_request_tuple = self._get_grpc_class(grpc_interface_module_list, _grpc_request_method_name.lower())
+            except Exception as e:
+                yield StatusMessage("failed to load the gRPC proto buff files : {}".format(e))
+                raise FunctionError(e)
+
+            # Converting resilient function input data into json object
+            try:
+                grpc_function_data = re.sub("'", '"', grpc_function_data)
+                grpc_function_json_data = json.loads(grpc_function_data)
+            except Exception as e:
+                raise FunctionError("Input Data must be in json formatted..!")
+
+            # Checking Communication type
+            if _grpc_communication_type.lower().strip() == "unary":
+                try:
+                    # Connecting to gRPC Server with given Authentication type
+                    stub_class_obj, grpc_channel_obj = self._gRPC_Connect_Server(_grpc_secure_connection, grpc_channel, \
+                                                                                 grpc_stub_tuple[1],
+                                                                                 _grpc_certificate_path)
+                    stub_method = self._get_method_from_stub_object(stub_class_obj, _grpc_communication_type,
+                                                                    stub_name=_grpc_rpc_stub_method_name)
+                    if grpc_function_json_data:
+                        if stub_method[1] is not None:
+                            if sys.version[0] == '2':
+                                # convert unicode strings to utf-8 for python 2.7
+                                import ast
```

Comment:
Move this import to the top and use `six` to check which version of Python is running

Reply:
ok